docs/release-notes.md

```diff
@@ -9,6 +9,7 @@ nav_order: 9
 ### Features
 
 - Validate merged/replaced Ignition configs if they are local/inline _(all base specifications)_
+- Added Kernel Argument rd.luks.keys in s390x layout sugar. 
```

Comment:
Wrong tense, always present not past

"Expand existing `CEX` sugar to include a required kernel arg"